rpc/jsonrpc/types/results.go

```diff
@@ -366,13 +366,14 @@ type ValidateAddressWalletResult = ValidateAddressResult
 
 // WalletInfoResult models the data returned from the walletinfo command.
 type WalletInfoResult struct {
-	DaemonConnected  bool    `json:"daemonconnected"`
-	Unlocked         bool    `json:"unlocked"`
-	CoinType         uint32  `json:"cointype,omitempty"`
-	TxFee            float64 `json:"txfee"`
-	TicketPurchasing bool    `json:"ticketpurchasing"`
-	VoteBits         uint16  `json:"votebits"`
-	VoteBitsExtended string  `json:"votebitsextended"`
-	VoteVersion      uint32  `json:"voteversion"`
-	Voting           bool    `json:"voting"`
+	DaemonConnected  bool              `json:"daemonconnected"`
+	Unlocked         bool              `json:"unlocked"`
+	CoinType         uint32            `json:"cointype,omitempty"`
+	TxFee            float64           `json:"txfee"`
+	TicketPurchasing bool              `json:"ticketpurchasing"`
```

Comment:
I noticed that this was never set. Remove?

Reply:
sure. it used to be, but is now unset after the ticketbuyer was moved out of the wallet package into its own repo, and now currently in the dcrwallet repo but under a different package.

this package in this module has not seen a stable release yet so it's safe to make breaking changes here as you've done.